mulhome_scripts/delete_all_circe.py

```diff
@@ -39,7 +47,8 @@ def delete_all_circe(app_name):
 
     logging.debug('Starting to teardown CIRCE')
     if jupiter_config.BOKEH == 3:
-        latency_file = '../stats/exp8_data/summary_latency/system_latency_N%d_M%d.log'%(len(node_list)+len(homes),len(dag))
+        latency_path = prepare_stat_path('../stats/')
```

Comment:
Can we move this to "stats/", a directory inside this folder?

Reply:
fixed